.editorconfig

```diff
@@ -1,7 +1,16 @@
+root = true
+
 [*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
 insert_final_newline = true
+trim_trailing_whitespace = true
 
 [*.py]
-charset = utf-8
-indent_style = space
 indent_size = 4
+max_line_length = 80
```

Comment:
In my experience this maximum is completely impractical. Let's leave no max for now.

Reply:
It is impractical, I agree, however it's the value currently used in the codebase and checked for in the flake8 configuration. The downstream branches increment this to 88, which is the default the `black` formatter uses.

Reply:
Fine...